bika/lims/api.py

```diff
@@ -1166,3 +1170,33 @@ def is_uid(uid, validate=False):
     if brains:
         assert (len(brains) == 1)
     return len(brains) > 0
+
+
+def is_date(date):
+    """Checks if the passed in value is a valid Zope's DateTime
+
+    :param date: The date to check
+    :type date: DateTime
+    :return: True if a valid date
+    :rtype: bool
+    """
+    if not date:
+        return False
+    return isinstance(date, DateTime) or isinstance(date, datetime)
+
+
+def get_date(value):
```

Comment:
Would you mind to change the name to `to_date` and pass as well a `default` value?